src/czml3/widget.py

```diff
@@ -0,0 +1,66 @@
+from .core import Document, Preamble
+
+CESIUM_TPL = """
+<link rel="stylesheet" href="https://cesium.com/downloads/cesiumjs/releases/{cesium_version}/Build/Cesium/Widgets/widgets.css" type="text/css">
+<div id="cesiumContainer" style="width:100%; height:100%;"><div>
+<script type="text/javascript">
+{script}
+</script>"""
+
+SCRIPT_TPL = """
+require.config({{
+    paths: {{
+        'cesium': 'https://cesium.com/downloads/cesiumjs/releases/{cesium_version}/Build/Cesium/Cesium'
+    }}
+}});
+
+require(['cesium'], function (dependency) {{
+    var czml = {czml};
+
+    var viewer = new Cesium.Viewer('cesiumContainer', {{
+        shouldAnimate : true
+    }});
+
+    // To have an inertial (ICRF) view
+    function icrf(scene, time) {{
```

Comment:
This function is included here for posterity but in fact it doesn't work 100 % correctly, see link below